### DISCUSSION TITLE
GNSS : Galiléo ?

### DISCUSSION THREAD

**Discussion by Pijos** [peterantypas/maiana#130] [Q&A]
The GNSS module L76-L is compatible with GPS, GLONASS, Galileo and Q-SS. But I only see GPS and Glonass satellites on my receiver?

**Comment by marcelrv** [comment]
I have the same experience. 
On my phone i can see many Galileo satellites, do they are in the visible area.

Does Galileo require different antenna?

**Comment by marcelrv** [comment]
Though the spec of GNSS L76-L indicates Galileo in the nmea spec L76-LB/L26-LB/LC86L is capable of accessing GPS, BeiDou and GLONASS systems. 

The nmea manual has
```
Either of the 
following three options are supported: 
 GPS only 
 GPS + GLONASS
 GPS + BeiDou
```
No mention of Galileo. 
Confusing....

**Comment by peterantypas** [comment]
I think I’m going to start looking for a new module in this form factor. Any suggestions?Sent from my iPhoneOn Jul 23, 2024, at 5:59 AM, Marcel ***@***.***> wrote:﻿
Though the spec of GNSS L76-L indicates Galileo in the nmea spec L76-LB/L26-LB/LC86L is capable of accessing GPS, BeiDou and GLONASS systems.
The nmea manual has
Either of the 
following three options are supported: 
 GPS only 
 GPS + GLONASS
 GPS + BeiDou

No mention of Galileo.
Confusing....

**Reply by hecko** [reply]
> I think I’m going to start looking for a new module in this form factor. Any suggestions?Sent from my iPhoneOn Jul 23, 2024, at 5:59 AM, Marcel ***@***.***> wrote:﻿ Though the spec of GNSS L76-L indicates Galileo in the nmea spec L76-LB/L26-LB/LC86L is capable of accessing GPS, BeiDou and GLONASS systems. The nmea manual has Either of the following three options are supported:  GPS only  GPS + GLONASS  GPS + BeiDou No mention of Galileo. Confusing.... —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

uBlox has a very good reputation in EU.

**Comment by marcelrv** [comment]
I think I know the answer . But asking anyway...
Does the firmware allow to pass commands to the gnss module? 

This link suggest  gallieo is available.. maybe it not well documented. If somehow I can pass through commands I can give it a try.

https://forums.quectel.com/t/l76-backup-mode/16709/4

https://forums.quectel.com/t/l76-gnss-search-mode-problem/10126

**Comment by peterantypas** [comment]
No, it's not and there are no GPIOs left to wire it either. The design will
have to change.
